_episodes/08-defensive.md

```diff
@@ -148,7 +158,8 @@ AssertionError                            Traceback (most recent call last)
 
 <ipython-input-20-408dc39f3915> in normalize_rectangle(rect)
       1 def normalize_rectangle(rect):
-      2     '''Normalizes a rectangle so that it is at the origin and 1.0 units long on its longest axis.'''
+      2     '''Normalizes a rectangle so that it is at the origin and 1.0 units long 
+      on its longest axis.'''
```

Comment:
Could you please re-run the above code cells (I think you need only two) to re-generate this error and replace it entirely?

Reply:
For simplicity, you can size your terminal to be 100-characters wide and copy here what you get.

Reply:
Looks like the script for the function has updated, so the line numbers in the error message were also updated and the long line issue avoided.